src/WhatToCook.Application/Infrastructure/Repositories/RecipesRepository.cs

```diff
@@ -41,16 +41,7 @@ public async Task Create(Recipe recipe)
 
     public async Task Delete(int id)
     {
-        var recipe = await _dbContext.Recipes.FindAsync(id);
-        if (recipe != null)
-        {
-            _dbContext.Recipes.Remove(recipe);
-            await _dbContext.SaveChangesAsync();
-        }
-        else
-        {
-            throw new NotFoundException($"Recipe with ID '{id}' not found.");
-        }
+        var recipe = await _dbContext.Recipes.FindAsync(id) ?? throw new NotFoundException($"Recipe with ID '{id}' not found.");
```

Comment:
Change log to use params or compile-time logging

Reply:
done, will be in the next commit